CONTRIBUTING.md

```diff
@@ -12,33 +12,39 @@ and other infrastructure for running OSGi-based systems.
 
 ## Developer resources
 
-Information regarding source code management, builds, coding standards, and
-more.
-
-* https://projects.eclipse.org/projects/eclipse.equinox/developer
+Information regarding source code management, builds, coding standards, and more.
 
 The project maintains the following source code repositories
 
-* https://github.com/eclipse-equinox/equinox.binaries
 * https://github.com/eclipse-equinox/equinox
+* https://github.com/eclipse-equinox/equinox.binaries
 * https://github.com/eclipse-equinox/p2
 
 This project uses GitHub to track ongoing development and issues.
 
-* Search for issues: https://github.com/eclipse-equinox/equinox.binaries/issues
+* Search for issues: https://github.com/eclipse-equinox/equinox/issues
 * Search for historical issues: https://eclipse.org/bugs/buglist.cgi?product=Equinox
-* Create a new report: https://github.com/eclipse-equinox/equinox.binaries/issues/new
+* Create a new report: https://github.com/eclipse-equinox/equinox/issues/new
 
 Be sure to search for existing bugs before you create another one. Remember that
 contributions are always welcome!
 
+### Setting up the Development Environment automatically, using the Eclipse Installer (Oomph)
+
+[![Create Eclipse Development Environment for Equinox](https://download.eclipse.org/oomph/www/setups/svg/Equinox.svg)](https://www.eclipse.org/setups/installer/?url=https://raw.githubusercontent.com/eclipse-equinox/equinox/master/equinox/org.eclipse.equinox.releng/EquinoxConfiguration.setup&show=true "Click to open Eclipse-Installer Auto Launch or drag into your running installer")
```

Comment:
@merks can you tell if there is already a manual for beginners how to use/apply an Oomph setup at a common location? I don't think it sustainable to repeat the steps in each repo, like it is for example done in m2e:
https://github.com/eclipse-m2e/m2e-core/blob/master/CONTRIBUTING.md#%EF%B8%8F-setting-up-the-development-environment-automatically-using-the-eclipse-installer-oomph

Reply:
Generally everyone's configuration has a link to this:

![image](https://github.com/eclipse-equinox/equinox/assets/208716/81b1e47e-9844-4d4e-ad70-0b7fcfa14589)

Yes, in the wiki I know.  😢 

https://wiki.eclipse.org/Eclipse_Platform_SDK_Provisioning

Indeed it's silly to repeat such instructions everywhere because they all follow the same pattern.

Reply:
Thanks for the hint!
I didn't saw it because the link in the buttons was wrong. 🤦🏽‍♂️ Good that this is fixed.

> Yes, in the wiki I know. 😢
> 
> https://wiki.eclipse.org/Eclipse_Platform_SDK_Provisioning

That's sufficient for now. In the future it would be nice to have the instructions for example right at that page in a sub-section or at a place with a better future? I can look into that at another time.